Oracle/shrine_AdapterMappings.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -x
+set -e
+SID=$1
+USERNAME=$2
+PASSWORD=$3
```

Comment:
passwords on the command line are visible to any user on the host. In many (most?) places we use environment variables, which are only visible to the user that owns the process.

I don't suppose we have made this a policy yet, though; I don't see documentation of it our wiki.

I wonder what Matt and/or Daryl think.

Reply:
Worth 15 minutes of our time to see if taking in the password environment variable works as opposed to the password itself.

Reply:
On the other hand it goes on the command line for sqlplus on line:17